yb-voyager/cmd/export.go

```diff
@@ -395,20 +399,27 @@ func checkDependenciesForExport() (binaryCheckIssues []string, err error) {
 			if err != nil {
 				return nil, err
 			} else if binaryCheckIssue != "" {
-
 				binaryCheckIssues = append(binaryCheckIssues, binaryCheckIssue)
 			}
 		}
-		if len(binaryCheckIssues) > 0 {
-			binaryCheckIssues = append(binaryCheckIssues, "Install or Add the required dependencies to PATH and try again\n")
-		}
 	}
 
 	if changeStreamingIsEnabled(exportType) || useDebezium {
+		// Check for java
```

Comment:
It's not ideal to be adding user messages ("install or add ... ") to the list of `binaryCheckIssues`. 
Nevertheless, for now: (we can pick that up later)
the else-ifs make it difficult to read IMO. 
```
javaIssue, err := checkJavaVersion()
if err != nil {
	return nil, err
} 
if javaIssue {
    		binaryCheckIssues = append(binaryCheckIssues, javaIssue)
 }
 if len(binaryCheckIssues) > 0 {
			binaryCheckIssues = append(binaryCheckIssues, "Install or Add the required dependencies to PATH and try again\n")

```
This should work, right?

Reply:
Yeah agreed. But in order to get the desired output format, this seemed like the easiest way. Also I missed the else condition on the external if:
```
else {
		if len(binaryCheckIssues) > 0 {
			binaryCheckIssues = append(binaryCheckIssues, "Install or Add the required dependencies to PATH and try again\n")
		}
	}
```

Reply:
Ah, this is getting a bit confusing to read. 

How about structure it this way : 
```
do pg checks, append to binaryCheckIssues
do java check, append to binaryCheckIssues (`if changeStreamingIsEnabled(exportType) || useDebezium {` block)
if binarchCheckIssues.len > 0: add that "Install or Add" message. 

Finally, do check for dbzm (again inside a `if changeStreamingIsEnabled(exportType) || useDebezium {` block) 
there you can directly append the "Please check your Voyager installation and try again" 
```

Reply:
Done